nft_ingester/src/config.rs

```diff
@@ -71,17 +71,15 @@ impl IngesterConfig {
     }
 
     pub fn get_account_backfill_stream_worker_count(&self) -> u32 {
-        self.account_backfill_stream_worker_count
-            .unwrap_or_else(|| self.get_account_stream_worker_count())
+        self.account_backfill_stream_worker_count.unwrap_or(0)
```

Comment:
Have to set this to zero for now because we are not configuring any backfill streams in the plugin.